www-misc/shellinabox-2.20-r4 fails to build

fails with error:
```
libhttp/httpconnection.c:224:14: error: static declaration of [1mstrcasestr follows non-static declaration
  224 | static char *strcasestr(const char *haystack, const char *needle) {
      |              ^~~~~~~~~~
In file included from libhttp/httpconnection.c:56:
/usr/include/string.h:364:14: note: previous declaration of [1mstrcasestr was here
  364 | extern char *strcasestr (const char *__haystack, const char *__needle)
      |              ^~~~~~~~~~
libhttp/httpconnection.c: In function [1mstrcasestr
libhttp/httpconnection.c:231:14: warning: return discards [1mconst qualifier from pointer target type [-Wdiscarded-qualifiers]
  231 |       return haystack;
      |              ^~~~~~~~
At top level:
libhttp/httpconnection.c:551:13: warning: [1maddHeader defined but not used [-Wunused-function]
  551 | static void addHeader(char **header, int *headerLength, const char *fmt, ...) {
      |             ^~~~~~~~~
libhttp/httpconnection.c:522:13: warning: [1mremoveHeader defined but not used [-Wunused-function]
  522 | static void removeHeader(char *header, int *headerLength, const char *id) {
      |             ^~~~~~~~~~~~
make[1]: *** [Makefile:764: libhttp/httpconnection.lo] Error 1
```
it compiles fine with -flto

--- a/sys-config/ltoize/files/package.cflags/lto.conf
+++ b/sys-config/ltoize/files/package.cflags/lto.conf
@@ -92,6 +92,7 @@ sys-libs/musl *FLAGS-=-flto*
 sys-process/criu *FLAGS-=-pipe* *FLAGS-=-flto* *FLAGS-="-fuse-linker-plugin" LDFLAGS-="-Wl,--hash-style=gnu" *FLAGS-="${GRAPHITE}" # Fewer settings may be possible. Needs testing. (Dependency of LXC.)
 sys-process/numactl *FLAGS-=-flto* # Undefined references
 www-apps/hugo *FLAGS-=-flto*
+www-misc/shellinabox *FLAGS-=-flto*
 x11-base/xorg-server *FLAGS-=-flto* # linking error during compilation
 x11-drivers/xf86-video-intel *FLAGS-=-flto*
 # END: lto

